5.1 Ex/5.1 Thats The Way.py

```diff
@@ -0,0 +1,35 @@
+"""Searching at given path files with given prefix, create a list of files name's
+and print it.
+"""
+
+from os import listdir
+
+
+def find_files_with_prefix(path: str, prefix: str) -> list:
+    """ Receives path to search in and prefix to search files with that prefix.
+    Return list of files name's that starting with the wanted prefix.
+    Note: I searched and find the solution at this link:
+    https://stackoverflow.com/questions/3964681/find-all-files-in-a-directory-with-extension-txt-in-python
+
+    @param path: For search at wanted path.
+    @param prefix: For search files with that prefix
+    @return: List of files name's that their prefix is as wanted that existing in wanted path.
+    """
+    return [file for file in listdir(path) if file.startswith(prefix)]
+
+
+def send_specific_data() -> list:
```

Comment:
I would replace this function with a partial that sets the prefix to deep, anyway, good thinking! the coupling to the specific path is redundant in my opinion